recipes-app/iot2050setup/files/iot2050setup.py

```diff
@@ -601,6 +608,55 @@ def selectTerminate(self):
             return default
         return 'on' if rdgroup.getSelection() else 'off'
 
+    def currentM2Select(self):
+        manual_config = subprocess.check_output("fw_printenv m2_manual_config",shell=True).decode('utf-8').lstrip().rstrip()
+        manual_config = manual_config.split("=")[1]
```

Comment:
I guess this would throw exception if `m2_manual_config` does not exist, isn't it?

Reply:
will not report an error。
when  manual_config is empty, we would get  empty value.